scene/gui/line_edit.cpp

```diff
@@ -45,6 +45,70 @@
 #include "editor/editor_settings.h"
 #endif
 
+void LineEdit::_edit() {
+	if (!is_inside_tree()) {
+		return;
+	}
+
+	if (!has_focus()) {
+		grab_focus();
+	}
+
+	if (!editable || editing) {
+		return;
+	}
+
+	editing = true;
+	_validate_caret_can_draw();
+
+	if (get_viewport()->get_window_id() != DisplayServer::INVALID_WINDOW_ID && DisplayServer::get_singleton()->has_feature(DisplayServer::FEATURE_IME)) {
```

Comment:
Not sure if this is an equivalent of the old code with window ID.

Reply:
`_edit` and `_unedit` was missing the new changes related to some bug fixes, they should follow the old focus entered/exited code. Thank you.